egs/swbd/s5c/local/chain/run_discriminative.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+set -e 
+set -o pipefail
+
+# this is run_discriminative.sh
+
+# This script does discriminative training on top of chain nnet3 system.
+# note: this relies on having a cluster that has plenty of CPUs as well as GPUs,
+# since the lattice generation runs in about real-time, so takes of the order of
+# 1000 hours of CPU time.
+# 
+. cmd.sh
+
+
+stage=0
+train_stage=-10 # can be used to start training in the middle.
+get_egs_stage=-10
+use_gpu=true  # for training
+cleanup=false  # run with --cleanup true --stage 6 to clean up (remove large things like denlats,
+               # alignments and degs).
+
+set -e
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+srcdir=exp/nnet3/nnet_ms_a
+train_data_dir=data/train_nodup_sp_hires
+online_ivector_dir=exp/nnet3/ivectors_train_nodup_sp
+degs_dir=
+lats_dir=
+
+## Objective options
+criterion=smbr
+one_silence_class=true
+
+dir=${srcdir}_${criterion}
+
+## Egs options
+frames_per_eg=150
+frames_overlap_per_eg=30
+truncate_deriv_weights=10
+
+## Nnet training options
+effective_learning_rate=0.0000125
+max_param_change=1
+num_jobs_nnet=4
+num_epochs=4
+regularization_opts=
+minibatch_size=64
+adjust_priors=true
```

Comment:
I don't think users will want to mess with adjust_priors and modify_learning_rates, so just hardcode them and don't use a variable.